less/pages/web-literacy.less

```diff
@@ -1,4 +1,11 @@
 .web-literacy {
+  .content-container {
+    margin-top: 1rem;
```

Comment:
Also, what is the purpose of this?

I'm thinking we should try flexbox to better center these, after talking with Kate.